retroshare.pri

```diff
@@ -653,6 +653,10 @@ android-* {
     }
     CONFIG *= no_libresapihttpserver
     CONFIG -= libresapihttpserver
+
+	CONFIG += no_rs_service_terminal_login
+	CONFIG += no_rs_service_webui_terminal_password
```

Comment:
this 2 line must be removed

Reply:
Android is the only platform where terminal login is not needed. So it's preferable to have terminal logins available by default and remove them only in android build, which is what I'm doing here.